rule-engine/rule-engine-components/src/test/java/org/thingsboard/rule/engine/telemetry/TbMsgAttributesNodeTest.java

```diff
@@ -69,12 +75,30 @@ class TbMsgAttributesNodeTest extends AbstractRuleNodeUpgradeTest {
     private TenantId tenantId;
     private DeviceId deviceId;
     private TbMsgAttributesNode node;
+    private TbMsgAttributesNodeConfiguration config;
+    private TbNodeConfiguration nodeConfiguration;
+    private TbContext ctx;
+    private RuleEngineTelemetryService telemetryService;
 
     @BeforeEach
-    void setUp() {
+    void setUp() throws TbNodeException {
         tenantId = new TenantId(UUID.fromString("6c18691e-4470-4766-9739-aface71d761f"));
         deviceId = new DeviceId(UUID.fromString("b66159d7-c77e-45e8-bb41-a8f557f434c1"));
         node = spy(TbMsgAttributesNode.class);
+        ctx = mock(TbContext.class);
+        config = new TbMsgAttributesNodeConfiguration().defaultConfiguration();
+        nodeConfiguration = new TbNodeConfiguration(JacksonUtil.valueToTree(config));
+        node.init(ctx, nodeConfiguration);
```

Comment:
Why you need to init in the setUp? I see that you do init almost in all your tests as well.